CHANGELOG.md

```diff
@@ -3,51 +3,81 @@ Note: This is in reverse chronological order, so newer entries are added to the
 Swift Next
 -----------
 
-* [#6111]
-
-  Package creation using `package init` now also supports the build tool plugin and command plugin types.
-
 * [#5728]
 
   In packages that specify resources using a future tools version, the generated resource bundle accessor will import `Foundation.Bundle` for its own implementation only. _Clients_ of such packages therefore no longer silently import `Foundation`, preventing inadvertent use of Foundation extensions to standard library APIs, which helps to avoid unexpected code size increases.
 
-* [#6185], [#6200]
-
-  Add a new `CompilerPluginSupport` module which contains the definition for macro targets. Macro targets allow authoring and distribution of custom Swift macros such as [expression macros](https://github.com/apple/swift-evolution/blob/main/proposals/0382-expression-macros.md).
 
 Swift 5.9
 -----------
 
+* [SE-0386]
+
+  SwiftPM packages can now use `package` as a new access modifier, allowing accessing symbols in another target / module within the same package without making it public.
+
+* [SE-0387]
+
+  New `swift experimental-sdk` experimental command is now available for managing Swift SDK bundles that follow the format described in [SE-0387]: "Swift SDKs for Cross-Compilation".
+
+* [SE-0391]
+
+  SwiftPM can now publish to a registry following the publishing spec as defined in [SE-0391]. SwiftPM also gains support for signed packages. Trust-on-first-use (TOFU) check which includes only fingerprints (e.g., checksums) previously has been extended to include signing identities, and it is enforced for source archives as well as package manifests.
+
 * [#5966]
 
   Plugin compilation can be influenced by using `-Xbuild-tools-swiftc` arguments in the SwiftPM command line. This is similar to the existing mechanism for influencing the manifest compilation using `-Xmanifest` arguments. Manifest compilation will also be influenced by `-Xbuild-tools-swiftc`, but only if no other `-Xmanifest` arguments are provided. Using `-Xmanifest` will show a deprecation message. `-Xmanifest` will be removed in the future.
 
+* [#6060]
+
+  Support for building plugin dependencies for the host when cross-compiling.
+
 * [#6067]
 
   Basic support for a new `.embedInCode` resource rule which allows embedding the contents of the resource into the executable code by generating a byte array, e.g.
 
   ```
   struct PackageResources {
-  static let best_txt: [UInt8] = [104,101,108,108,111,32,119,111,114,108,100,10]
+    static let best_txt: [UInt8] = [104,101,108,108,111,32,119,111,114,108,100,10]
```

Comment:
Hm, why is a space here? It definitely won't be there in the generated code :)

Reply:
just to improve formatting?